Rakefile

```diff
@@ -6,6 +6,26 @@ task :dev do
 end
 
 
+desc 'Build the site'
+task :build do
+  sh 'bundle exec jekyll build'
```

Comment:
do we need to use something like jekyll watch for the new site or will running the rake command rebuild it using exec and relaunch the page.